python/taichi/misc/error.py

```diff
@@ -1,47 +0,0 @@
-import sys
-import traceback
-
-from colorama import Fore
-
-
-def enable_excepthook():
```

Comment:
Not sure if we definitely want to remove this. If we finally decide to remove this, we can remove all `_taichi_skip_traceback` in the codebase and all docs or configs related to this feature (https://github.com/taichi-dev/taichi/search?q=excepthook).

As we have @lin-hitonami 's new error report mechanism now, do we still want to keep the hacky pretty printing of the traceback stack? @k-ye

Reply:
I'm fully down to remove _taichi_skip_traceback :D

Reply:
+1, https://github.com/taichi-dev/taichi/issues/2489